cobspec/src/main/java/com/saralein/cobspec/data/Log.java

```diff
@@ -0,0 +1,21 @@
+package com.saralein.cobspec.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Log implements LogStore {
```

Comment:
Do we need a `LogStore` yet. It looks like this is the only implementation.

Reply:
Deinterfaced this.